server.py

```diff
@@ -152,12 +175,14 @@ def get_product_data():
         reader = csv.DictReader(csvfile)
         for row in reader:
             prod_data[int(row['id'])] = {   'id':int(row['id']), 'name':row['name'], 'category':row['category'], \
-                                            'discontinued': row.get('discontinued', False)   }
+                                            'discontinued': row.get('discontinued', False), \
+                                            'price': row['price']   }
```

Comment:
Perhaps price should be casted into its appropriate data-type here, e.g. int or decimal. By default, it will assume a string data-type, which leaves us the work of casting it into a numeric value - elsewhere, e.g. line 59.